### ISSUE TITLE
Different issue with SciBite CORD transform

### ISSUE THREAD

**Issue by justaddcoffee** [Knowledge-Graph-Hub/kg-covid-19#89]
Fresh out of the oven from Jenkins: 

```
09:03:59  [transform.py][           transform] INFO: Parsing ScibiteCordTransform
09:06:35  Traceback (most recent call last):
09:06:35    File "run.py", line 81, in <module>
09:06:35      cli()
09:06:35    File "/var/lib/jenkins/workspace/ge_Graph_Hub_kg-covid-19_jenkins/config/venv/lib/python3.7/site-packages/click/core.py", line 829, in __call__
09:06:35      return self.main(*args, **kwargs)
09:06:35    File "/var/lib/jenkins/workspace/ge_Graph_Hub_kg-covid-19_jenkins/config/venv/lib/python3.7/site-packages/click/core.py", line 782, in main
09:06:35      rv = self.invoke(ctx)
09:06:35    File "/var/lib/jenkins/workspace/ge_Graph_Hub_kg-covid-19_jenkins/config/venv/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
09:06:35      return _process_result(sub_ctx.command.invoke(sub_ctx))
09:06:35    File "/var/lib/jenkins/workspace/ge_Graph_Hub_kg-covid-19_jenkins/config/venv/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
09:06:35      return ctx.invoke(self.callback, **ctx.params)
09:06:35    File "/var/lib/jenkins/workspace/ge_Graph_Hub_kg-covid-19_jenkins/config/venv/lib/python3.7/site-packages/click/core.py", line 610, in invoke
09:06:35      return callback(*args, **kwargs)
09:06:35    File "run.py", line 59, in transform
09:06:35      kg_transform(*args, **kwargs)
09:06:35    File "/var/lib/jenkins/workspace/ge_Graph_Hub_kg-covid-19_jenkins/config/kg_covid_19/transform.py", line 51, in transform
09:06:35      t.run()
09:06:35    File "/var/lib/jenkins/workspace/ge_Graph_Hub_kg-covid-19_jenkins/config/kg_covid_19/transform_utils/scibite_cord/scibite_cord.py", line 46, in run
09:06:35      self.parse_annotations(node_handle, edge_handle, data_files[0])
09:06:35    File "/var/lib/jenkins/workspace/ge_Graph_Hub_kg-covid-19_jenkins/config/kg_covid_19/transform_utils/scibite_cord/scibite_cord.py", line 67, in parse_annotations
09:06:35      for filename in os.listdir(data_dir):
09:06:35  FileNotFoundError: [Errno 2] No such file or directory: 'data/raw/data/biorxiv_medrxiv/biorxiv_medrxiv'
```

Possibly some simple file housekeeping issue

**Comment by justaddcoffee** [comment]
@deepakunni3 possibly something changed between version 1.2 -> 1.3 in this `CORD-19_1_X.zip`?

**Comment by deepakunni3** [comment]
Yes, the path of the extracted folders have changed slightly.

PR incoming...

**Comment by justaddcoffee** [comment]
Closed by #94
